login.php

```diff
@@ -41,6 +41,8 @@
                 $_SESSION['user_role'] = $result[0]['user_role'];
                 $_SESSION['toggle_summary'] = $result[0]['toggle_summary'];
 
+                $_SESSION['login_message'] = "Welcome, " . $_SESSION['username'] . "";
```

Comment:
The `. ""` at the end of the string is unnecessary. You can end the variable assignment at: `"Welcome, " . $_SESSION['username']`